docs/aws-openedx/aws-deployment.md

```diff
@@ -0,0 +1,36 @@
+# Open edX AWS Production Deployment and Upgrades
+
+In contrast to OCIM updates, which are automated through a control panel, AWS updates are more manual and we have to actually log in to the AWS panel and SSH to the instance.
+Apart from this, the main part of the deployment process is an `ansible` playbook which does all the provisioning automatically.
+
+Some of the components we use in Amazon are:
+- We always create *two* instances: the one to be provisioned, and a proxy one, called [director](../shared/director.md), that we use as proxy to connect to the final one.
+- We store the data in an [RDS](../shared/RDS.md) instance, that we create and backup separately
+- At the end of the deployment process, we switch an IP to point to the final instance; in Amazon it's called [Elastic IP](../shared/Elastic_IP.md)
+
+This article is about deploying LMS/CMS.
+See also how to setup [Analytics in AWS](../analytics/AWS_setup.md).
+
+## Initial deployment
+Similar to upgrades (see below).
+
+## How to upgrade an instance
+
+The basics are:
+
+1. set up `vars.yml` in `edx-configuration`
```

Comment:
1. It doesn't matter where vars.yml lives; its path is set when calling by the `ansible-playbook` command.
1. Note that the variables in this file override the defaults provided by [edx:configuration](https://github.com/edx/configuration)
1. Secure information, e.g. secret keys and OAuth keys, should be kept safe and private.